dali/nvjpeg/nvjpeg_helper.cc

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "dali/operators/decoder/nvjpeg/nvjpeg_helper.h"
+#include "dali/nvjpeg/nvjpeg_helper.h"
```

Comment:
I don't like the idea of this file being moved - there's a copy in imgcodec - it was intentional; now I'm afraid we might see some conflicts (it only works because we made nvjpeg_helper.h header-only).

Reply:
Done, I only move the wrapper and stub placement.